database/migrations/20220110072026_seekers.js

```diff
@@ -0,0 +1,21 @@
+exports.up = async function (knex) {
+  await knex.raw(`create extension if not exists "uuid-ossp"`);
+  return await knex.schema.createTable("seekers", function (table) {
+    table
+      .uuid("uuid")
+      .notNullable()
+      .primary()
+      .defaultTo(knex.raw("uuid_generate_v4()"));
+    table.string("user_uuid", 255).notNullable();
+    table.string("name", 255).notNullable();
+    table.jsonb("seeker_description").nullable();
```

Comment:
I think we should enforce the user to enter `seeker_description`, `seeker_description`, and  `job_preferences` and hence they should not be nullable. As the whole logic depends on these columns. @mahesh-muttinti

Reply:
ok